Library/Homebrew/service.rb

```diff
@@ -355,6 +355,23 @@ def command
       @run&.map(&:to_s)
     end
 
+    sig { params(value: T.nilable(Integer)).returns(T.nilable(Integer)) }
+    def nice(value = nil)
+      case T.unsafe(value)
+      when nil
+        @nice
+      when Integer
+        if (T.must(value) < -20) || (T.must(value) > 19)
```

Comment:
```suggestion
        unless (-20..19).cover?(nice)
```
This is concise, and also lets you avoid the `T.must`s. (You could even extract the magic numbers into a const, e.g. `NICE_RANGE = -20..19`)

Reply:
Could you elaborate on the issue in using `NICE_RANGE` in tests?